etc/FindMessages.tcl

```diff
@@ -13,7 +13,7 @@ exec tclsh $0 ${1+"$@"}
 # Find warning/error message IDs and detect collisions.
 # Usage: FindMessages.tcl > doc/messages.txt
 
-proc scan_file { file warn_regexp } {
+proc scan_file { file warn_regexp is_filter_objs } {
```

Comment:
uh, no. that is too specific. I'd rather see the same error messages used by all the filter_objs callers